src/hera/workflows/io/_io_mixins.py

```diff
@@ -175,7 +206,31 @@ def __init__(self, /, **kwargs):
         super().__init__(**kwargs)
 
     @classmethod
-    def _get_outputs(cls) -> List[Union[Artifact, Parameter]]:
+    def path(cls, attribute: Any) -> Path:
```

Comment:
This is not used anywhere right? Do we need this?

Reply:
Ah yep, will move to #1098